src/roiextractors/testing.py

```diff
@@ -11,6 +11,29 @@
 inttype = (int, np.integer)
 
 
+import numpy as np
+
+from roiextractors import NumpyImagingExtractor
+
+
+def generate_dummy_imaging_extractor(
+    num_frames: int = 100,
+    rows: int = 10,
+    columns: int = 10,
+    num_channels: int = 1,
+    sampling_frequency: float = 30000,
+    dtype="int16",
+):
+    channel_names = [f"channel_num_{num}" for num in range(num_channels)]
+    video = np.random.randint(low=0, high=256, size=(num_frames, rows, columns, num_channels)).astype(dtype)
```

Comment:
I think that's a good idea that, the only thing that I am not so sure about is the random values for the float case but I don't really have any other suggestion.

Reply:
Oh I meant to mention, the random values in the float case are uniformly distrbuted which is good because then if someone wants to run some kind of targetted test on specific ranges of data values (which onyl they would know) they can easily to dhat by shift/scale modification of the default uniform values.

*I.e.*, `video_with_other_values = video * (b - a) + a` will now uniformly range any interval `[a, b]`.

Of course if they want a different distribution altogether, then they can either just make their own toy data or use the inverse transform method `video_with_new_distribution = inverse_cumulative_distribution(video)`.